use-cases/kitchen-sink.md

```diff
@@ -48,3 +48,7 @@ sgMail
   .then(() => console.log('Mail sent successfully'))
   .catch(error => console.error(error.toString()));
 ```
+
+### Error Handling:
```

Comment:
Not a big deal but perhaps this title should be `Caveats`?

Reply:
agreed @reedsa , I'll update the PR